awx/main/tasks.py

```diff
@@ -112,6 +112,7 @@
 
 from rest_framework.exceptions import PermissionDenied
 
+import sdb
```

Comment:
you'll want to remove the sdb trace here

Reply:
Yup , i am gonna remove the SDB commands.